static/js/recent_topics.js

```diff
@@ -551,6 +551,14 @@ function is_focus_at_last_table_row() {
     return row_focus === topic_rows.length - 1;
 }
 
+export function focus_clicked_element(e, col) {
+    current_focus_elem = "table";
+    col_focus = col;
+
+    const $elem = $(e.target);
+    row_focus = $elem.closest("tr").index();
+}
```

Comment:
It might be cleaner to pass `$(e.target)` to this function, and have it just take a `$elt`, just from an abstraction boundary perspective.

(We generally try to avoid passing browser event objects between modules)

Reply:
Done.